apps/07jquery-tic-tac-toe/index.html

```diff
@@ -22,7 +22,8 @@
     <div data-cell="8"></div>
   </div>
   <div id="announce-winner"></div>
-  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <button id="clear">Clear Board</button>
+  <script type="text/javascript" src="../vendor/jquery.js"></script>
```

Comment:
Good catch, let's just bring in jquery into the repo then. It will at least insulate us if the internet ever gets spotty.